docs/topics/http-api-wf-swarm-pom.adoc

```diff
@@ -0,0 +1,172 @@
+      <artifactId>arquillian-junit-container</artifactId> <7>
+      <version>1.1.12.Final</version>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.wildfly.swarm</groupId>
+        <artifactId>wildfly-swarm-plugin</artifactId> <8>
+        <version>${version.wildfly.swarm}</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>package</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <artifactId>maven-war-plugin</artifactId>
+        <configuration>
+          <failOnMissingWebXml>false</failOnMissingWebXml>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+
+  <profiles>
+    <profile>
+      <id>openshift</id> <9>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>io.fabric8</groupId>
+            <artifactId>fabric8-maven-plugin</artifactId>
+            <executions>
+              <execution>
+                <goals>
+                  <goal>resource</goal>
+                  <goal>build</goal>
+                </goals>
+              </execution>
+            </executions>
+            <configuration>
+              <generator>
+                <includes>
+                  <include>wildfly-swarm</include>
+                </includes>
+                <excludes>
+                  <exclude>webapp</exclude>
+                </excludes>
+              </generator>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+    <profile>
+      <id>openshift-it</id> <10>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-failsafe-plugin</artifactId>
+            <executions>
+              <execution>
+                <goals>
+                  <goal>integration-test</goal>
+                  <goal>verify</goal>
+                </goals>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+  </profiles>
+</project>
+----
+
+<1> link:https://github.com/openshiftio/booster-parent/blob/master/pom.xml[Parent pom] which contains the necessary dependencies and the required versions for the booster projects.
+<2> These are properties for setting the {WildFlySwarm} and RESTEasy version.
+<3> link:https://github.com/wildfly-swarm/wildfly-swarm/blob/master/boms/bom/pom.xml[Parent Bom] for building {WildFlySwarm} applications with maven.
+<4> link:https://jcp.org/en/jsr/detail?id=339[JAX-RS] dependency for creating REST services.
+<5> link:https://docs.jboss.org/resteasy/docs/3.1.3.Final/userguide/html_single/#RESTEasy_Client_Framework[RESTEasy client] dependency for connecting to other REST services which is used in testing.
+<6> link:http://arquillian.org[Arquillian] dependency for testing.
```

Comment:
Again this is actually a Swarm fraction. One that brings an Arquillian adapter for Swarm.